### PR TITLE
feat(gatsby-theme): added fallback for webp image presets and allowed to override default sharp properties

### PR DESCRIPTION
**Pull request by beliayeu**
## Changes
* disallowed uploading images in webp format
* added fallback for fluid webp images
* exposed fluid webp sharp args, site builders can adjust it in site gatsby-config.js

### CODE REVIEW

**Review by VancheeZze — DISMISSED** [review]

**Review by VancheeZze — DISMISSED** [review]

File: packages/gatsby-theme-bodiless/README.md [hunk 1]
```diff
@@ -9,6 +9,30 @@ const gatsbyThemeBodiless = require('@bodiless/gatsby-theme-bodiless');
 
 // TODO: DEMONSTRATE API
 ```
+
```
Comment (hvanyo) [inline packages/gatsby-theme-bodiless/README.md:12]: Can we move and/or copy this documentation to https://johnsonandjohnson.github.io/Bodiless-JS/#/Development/Guides/BuildingSites/ImageConfigurations

File: packages/bodiless-components/src/Image.tsx [hunk 2]
```diff
@@ -128,7 +128,7 @@ export function DropZonePlugin({ formApi, targetFieldName, ui }: {
 
   const { getRootProps, getInputProps, isDragReject } = useDropzone({
     onDrop,
-    accept: 'image/jpeg, image/png, image/svg+xml, image/gif, image/webp, image/apng',
+    accept: 'image/jpeg, image/png, image/svg+xml, image/gif, image/apng',
```
Comment (hvanyo) [inline packages/bodiless-components/src/Image.tsx:131]: On test can we remove, the two cases where we demonstrate the use of webp -> "Delete the two webp cases in Editable images & Linkable Editable images section"

File: packages/gatsby-theme-bodiless/README.md (hunk 1 above)
Comment (beliayeu) [reply packages/gatsby-theme-bodiless/README.md:12]: fixed in 028c8c282a6a02ca8c061c75695c6cabb54bca8b

File: packages/bodiless-components/src/Image.tsx (hunk 2 above)
Comment (beliayeu) [reply packages/bodiless-components/src/Image.tsx:131]: fixed in 028c8c282a6a02ca8c061c75695c6cabb54bca8b

**Review by VancheeZze — APPROVED** [review]

**Review by hvanyo — APPROVED** [review]

**Pull request merged**